doc/cephfs/administration.rst

```diff
@@ -94,12 +94,13 @@ available, then the "failed" daemon will return as a standby.
 
 ::
 
-    mds deactivate <role>
+    mds deactivate [<fs_name>:]<rank>
```

Comment:
The `role` argument can actually be other forms too;

https://github.com/jan--f/ceph/blob/2155f7fef6415ac0cca88e1eb5a7b12c995f3433/src/mds/FSMap.cc#L970-L982

I'd prefer leaving it as role and link to a doc explaining what a role's form can be.